File: templates/java/zh/text/code/beeperLineBug1.java
/*
 * 文件：锥体行Bug.java
 * -------------
 * 使用while循环放置一条锥体 s的行。该程序适用
 * 于任何规模的世界。然而，因为每个世界需要一个较少move总比putBeeper的IT总是错过了锥体 。
 */
import stanford.卡雷尔.*;
public class 锥体行Bug extends 卡雷尔 {
   public void run() {
   	  /* 重复，直到卡雷尔面临一堵墙 */
      while(frontIsClear()) {
      	 /* 在当前的广场上放置一个锥体 */ 
         putBeeper();
      	 /* move到下一个广场 */
         move();
      }
   }
}

File: templates/python/zh/text/code/beeperLineBug1.py
＃文件：BeeperLineBug.py
＃------------------------------
＃使用while循环放置一行锥体 s。
＃此程序适用于任何规模的世界。
＃但是，因为每个世界都减少了一个
＃ move比put_ 锥体总是丢失锥体 。
from 卡雷尔.stanford import *

＃程序从main开始
def main():
   ＃重复直到karel面对墙
   while front_is_clear():
      ＃在当前正方形上放置一个锥体
      放锥体()
      ＃ move下一个方块
      move()

